sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -1,16 +1,7 @@
 # Release History
 
-## 1.1.0b2 (Unreleased)
-
-### Features Added
-- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-- The `MicrosoftBotIdentifier` and `MicrosoftBotProperties` have been deprecated.
+## 1.1.0 (2024-04-15)
+General Availability version of PSTN dial-out capability feature
```

Comment:
Nit: please add a dot at the end. "General Availability version of PSTN dial-out capability feature.". I am going to make changes to .NET SDK as well.

Reply:
done thanks